test/analysis/max_min_driving_force.jl

```diff
@@ -0,0 +1,121 @@
+@testset "Max min driving force analysis" begin
```

Comment:
this is a bit too much copypastish from the docs; can we have a realistic model?

The point: this test is now irreproducible, either because others won't know how to write it again or because base data is not sourced.

Reply:
so this is actually a realistic model using realistic data for glycolysis (the EMP pathway to be exact). It is from the paper cited in the docstrings, although I use more updated thermo data. I used it in both because it should be recognizable to thermo people :) Also, I trust the result, hence it being included in the test

Reply:
That doesn't help the reproducibility though. Would other people write the model in _precisely_ the same way you did?